zenodo/config.py

```diff
@@ -1047,6 +1053,12 @@ def _(x):
         'schema': 'http://schema.datacite.org/meta/kernel-3/metadata.xsd',
         'serializer': 'zenodo.modules.records.serializers.oaipmh_oai_datacite',
     },
+    'oai_datacite4': {
```

Comment:
@lnielsen should we change this to be the default `oai_datacite` format as well?